controllers/dashboard_test.go

```diff
@@ -163,6 +163,35 @@ var _ = Describe("DashboardController", func() {
 					Expect(len(matches)).To(BeZero())
 				})
 			})
+
+			Context("given `keyword` param is set", func() {
+				It("shows only `keyword` filter records", func() {
+					keywordFilter := "keyword_keyword_keyword"
```

Comment:
To make this test easier to understand, I recommend the following naming:

```suggestion
					keywordFilter := "expected_keyword"
```